autopush/websocket.py

```diff
@@ -1215,6 +1215,8 @@ def process_register(self, data):
             d = self.deferToThread(self.conf.make_endpoint,
                                    self.ps.uaid, chid, data.get("key"))
         else:
+            if self.conf.disable_simplepush:
+                return self.bad_message("register", "No longer supported")
```

Comment:
We should have. This ensures that any older clients do not get a URL. Previously, it was possible that this could occur.